app/routes/reminders.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model(){
+    return [1, 2, 3, 4, 5]
```

Comment:
Because we're using Mirage we can make a query to the store to findAll records. Check out [the docs](https://guides.emberjs.com/v2.11.0/models/finding-records/#toc_retrieving-multiple-records).